device-types/Juniper/PTX10001-36MR.yaml

```diff
@@ -0,0 +1,88 @@
+---
+manufacturer: Juniper
+model: PTX10001-36MR
+slug: ptx10001-36mr
```

Comment:
Please append the manufacturer name to the device like so: `juniper-`

Reply:
sorry, I don't get it. Where am I supposed to write "juniper-"?
The other juniper in the repo are reported like this:

---
manufacturer: Juniper
model: QFX5110-48S-4C
slug: qfx5110-48s-4c
interfaces:
...

---
manufacturer: Juniper
model: MX480
slug: mx480
u_height: 8
is_full_depth: true
...

Consider that I used the QFX file as a template for my ptx.

Reply:
That means they havent been migrated yet, one moment ill take care of that now

Reply:
@gpipito please run a git pull and take a look at the other Juniper devices now. You should see the new slug naming convention